src/Components/DefLinear/DefLinear.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import './DefLinear.css';
+import Button from '@material-ui/core/Button';
+import {useHistory} from 'react-router-dom';
+
+function DefLinear(props) {
+    const history=useHistory()
+    const onBackClick = () => {
+      history.push("/");
+    }
+    const onNextClick = () => {
+        history.push("/LinearGoal");
+    }
```

Comment:
Not something to correct - I just wanted to say I like the use of hooks here! While normally I'd put history pushes in arrow functions bound to buttons, I think it makes sense to create functions here since you or one of your teammates may change the paths down the line.